.github/scripts/trymerge.py

```diff
@@ -426,21 +451,29 @@ def get_changed_files(self) -> List[str]:
             raise RuntimeError("Changed file count mismatch")
         return self.changed_files
 
-    def _get_reviewers(self) -> List[Tuple[str, str]]:
-        reviews_count = int(self.info["reviews"]["totalCount"])
-        nodes = self.info["reviews"]["nodes"]
-        if len(nodes) != reviews_count:
-            raise RuntimeError("Can't fetch all PR reviews")
+    def _get_reviews(self) -> List[Tuple[str, str]]:
+        if self._reviews is None:
+            self._reviews = []
+            info = self.info
+            for _ in range(100):
```

Comment:
this means we can now handle 100*100 reviews on a PR.

Reply:
Yeah, 10k comments sounds more than reasonable